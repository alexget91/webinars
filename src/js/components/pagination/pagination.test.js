import React from "react";
import renderer from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";
import Pagination from "./pagination";

it(`Pagination correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<MemoryRouter>
      <Pagination
        elementsCount={2}
        elementsPerPage={1}
        currentPage={1}
        cssClass={`cssClass`}
      />
    </MemoryRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
