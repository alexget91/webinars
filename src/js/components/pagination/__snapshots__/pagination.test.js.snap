// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination correctly renders after relaunch 1`] = `
<div
  className="pagination cssClass"
>
  <a
    className="pagination__link active"
    href="/?page=1"
    onClick={[Function]}
  >
    1
  </a>
  <a
    className="pagination__link"
    href="/?page=2"
    onClick={[Function]}
  >
    2
  </a>
</div>
`;
